.github/workflows/semantic-pull-requests.yml

```diff
@@ -9,13 +9,39 @@ on:
       - synchronize
 
 permissions:
-  pull-requests: read
+  pull-requests: write
 
 jobs:
   validate-pr:
     name: Validate PR title
     runs-on: ubuntu-latest
     steps:
       - uses: amannn/action-semantic-pull-request@v5
+        id: lint_pr_title
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
\ No newline at end of file
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - uses: marocchino/sticky-pull-request-comment@v2
+        # When the previous steps fails, the workflow would stop. By adding this
+        # condition you can continue the execution with the populated error message.
+        if: always() && (steps.lint_pr_title.outputs.error_message != null)
```

Comment:
Do we actually need this `always()`?

Reply:
They used it in the action [example](https://github.com/amannn/action-semantic-pull-request#outputs), I think it's been used to handle[ all cases ](https://docs.github.com/en/actions/learn-github-actions/expressions#always)even if it's failed/canceled.

Reply:
Interesting. Ok!